app/ClientApp/src/components/MessageProps.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import MessageBox from './MessageBox';
+
+document.getElementById('NAME').innerHTML = 'Name';
+const date = new Date();
+document.getElementById('DATE').innerHTML = date.getHours();
+document.getElementById('TEXT').innerHTML = 'labas';
```

Comment:
These should be parameters of the function and their value should be written inline